src/routes/landing/components/FilterableCitySelector.js

```diff
@@ -43,48 +49,41 @@ class FilterableCitySelector extends React.Component<PropsType, StateType> {
       Geolocation.requestAuthorization()
     }
     Geolocation.getCurrentPosition(
-      position => {
-        const currentLongitude = position.coords.longitude
-        const currentLatitude = position.coords.latitude
-        this.setState({ currentLongitude: currentLongitude })
-        this.setState({ currentLatitude: currentLatitude })
-      },
-      error => {
-        alert(error.message)
-      },
+      position => this.setState({
+        currentLongitude: position.coords.longitude,
+        currentLatitude: position.coords.latitude
+      }),
+      alert(this.props.t('locationError')),
```

Comment:
I think the lambda `error =>` is missing here. Else this is executed always.

Reply:
Minor: I think it is better to use the Alert API of RN: https://facebook.github.io/react-native/docs/alert